app/helpers/research_sessions_helper.rb

```diff
@@ -1,10 +1,14 @@
 module ResearchSessionsHelper
   def current_step_params
     component_params(*flat_params(step))
+      .merge(unvarying_params(step))
+      .merge(able_to_consent: false)
```

Comment:
If we have `able_to_consent?` as a convenience method I think we should use it consistently and remove this hardcoded false.

Reply:
After looking more at the context in which this is called, I'm going to leave this as false as it is more intent revealing and keeps things consistent (in case able_to_consent was set against the model before going back to a step).